variables.tf

```diff
@@ -73,3 +73,8 @@ variable "force_destroy" {
   description = "A boolean that indicates all objects should be deleted from the bucket so that the bucket can be destroyed without errors. These objects are not recoverable"
   default     = false
 }
+
+variable "domain_enabled" {
+  default     = "true"
+  description = "Set to false to prevent the module from creating a DNS Zone for the kops domain"
```

Comment:
Should we conform with the other boolean descriptions here? I.e. "A boolean that determines whether a DNS Zone for the kops domain is created."